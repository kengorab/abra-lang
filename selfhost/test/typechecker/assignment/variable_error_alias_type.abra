type Foo {}

Foo = 123