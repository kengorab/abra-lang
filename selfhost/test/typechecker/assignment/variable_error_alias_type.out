Error at %FILE_NAME%:3:1
Cannot assign to variable 'Foo'
  |  Foo = 123
     ^
'Foo' is a type, which cannot be overwritten
